index.html

```diff
@@ -1,2002 +1,2304 @@
-			<a href="" id="mobile-menu-button"><img src="images/menu.svg" alt="Menu"></a>
-		</header>
-		<div id="mobile-menu">
-			<ul>
-				<li><a href="" id="nav-welcome"><span>01: Welcome</span></a></li>
-				<li><a href="" id="nav-stories"><span>02: Stories with Impact</span></a></li>
-				<li><a href="" id="nav-leadership"><span>03: Active Leadership</span></a></li>
-				<li><a href="" id="nav-talent"><span>04: Investing in Talent</span></a></li>
-				<li><a href="" id="nav-support"><span>05: Donor Support</span></a></li>
-				<li><a href="" id="nav-press"><span>06: Awards & Press</span></a></li>
-				<li><a href="" id="nav-financials"><span>07: Financials</span></a></li>
-			</ul>
-		</div>
-		<div id="pages">
-
-			<!-- 01: Welcome -->
 
-			<div class="page open" id="page-welcome">
-				<div class="page-content">
-					<p class="page-label">01: Welcome</p>
-					<section class="hero">
-						<div class="bg"></div>
-						<div class="inner">
-							<p class="logo"><a href=""><img src="images/logo.svg" alt="PRX"></a></p>
-							<p class="expand"><img src="images/expand-the-story.svg" alt="Expand the Story"></p>
-							<h1>Annual Report 2021</h1>
-
-							<div class="audio">
-								<audio preload="auto" controls>
-									<source src="audio/audio.mp3">
-								</audio>
-								<script src="js/audio-player.js"></script>
-								<script>
-									$(function() {
-										$('audio').audioPlayer();
-									});
-								</script>
-							</div>
-							<p class="body">
-								PRX believes in the power of audio storytelling. Experiencing new stories expands our empathy, changes our minds, and transforms our communities.
-								As a home for innovation in audio, the work and culture of PRX reflect our mission: to lead the expansion of public media as a trusted source of
-								knowledge, information, and inspiration for all. In 2021 we broke new ground in this effort, together with our dedicated listeners, creators, and
-								supporters. Thank you.
+<head>
+	<title>PRX | Expand the Story</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="css/prx.css?2022-03-03">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link
+		href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,800;1,300;1,400;1,800&display=swap"
+		rel="stylesheet">
+	<link rel="icon" type="image/png" href="https://media.prx.org/favicon-16x16.png" sizes="16x16">
+	<link rel="icon" type="image/png" href="https://media.prx.org/favicon-32x32.png" sizes="32x32">
+	<script src="https://code.jquery.com/jquery-3.6.0.min.js"
+		integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+	<script defer data-domain="2021.prx.org" src="https://plausible.io/js/plausible.js"></script>
```

Comment:
Here's the code. My code editor reindented everything, but this is the only change.